.github/workflows/release.yml

```diff
@@ -48,7 +48,13 @@ jobs:
         id: release
         run: |
           currentTag=$(git describe --abbrev=0 --tags)
-          echo "current release tag ${currentTag}"
+          echo "Current release tag ${currentTag}"
+
+          echo ${currentTag}  | grep -q "[0-9]\+\.[0-9]\+\.[0-9]\+"
+          if [ $? -ne 0 ]; then
+            echo "Current tag format won't let us compute the new tag name. Required format v[0-9].[0-9].[0-9]"
```

Comment:
In prebid Cache we don't need the `v` prefix. Can we please reword to `Required format [0-9]+.[0-9]+.[0-9]+`?

Reply:
Going forward we are planning to add `v` to all releases. 

For example if last release tag is 0.25.0 then next tag will have `v` prefix i.e `v0.25.0`. This is to get more inline with go module tag format recommendations. 

Current changes are not supporting above requirement. Will update script and push commit

Reply:
@guscarreon  changes are made in [58615b4](https://github.com/prebid/prebid-cache/pull/135/commits/58615b4766a255a3d2883beb245c89d43cb07946) to add prefix 'v'

case:  previous release has no 'v' prefix
<img width="200" height="98" alt="Screenshot 2023-03-10 at 12 41 59 AM" src="https://user-images.githubusercontent.com/24757781/224132273-bc1b08ea-e645-447c-b3c6-0bf9f5b94631.png"> 

case: previous release has 'v' prefix
<img width="200" height="98" alt="Screenshot 2023-03-10 at 12 44 30 AM" src="https://user-images.githubusercontent.com/24757781/224132328-09b40b5b-4cba-475f-a0a0-bb6d4b9e8399.png">